src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">$(OS)_Debug</Configuration>
+    <Configuration Condition="'$(Configuration)' == '' ">Windows_Debug</Configuration>
```

Comment:
So this was more of a revert that anything else...the Sockets project did not build so I reverted the file. My immediate guess is that the $(OS) tag is set to either Windows or Unix, then when we include dir.props, some transformations occur to set the TargetsWindows or TargetsUnix flags; however, since Sockets is Linux, OSX, or Windows, that caused problems since there isn't a Unix option for the sources. 

Just my first guess :)